test/bdd/features/vc_dev_api.feature

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@vc_dev_api
+Feature: Request DID Config
+  Scenario: Request DID Config for Issuer with ldp
+    When I request did config for "issuer" with ID "i_myprofile_ud_P256k1" and type "ldp"
```

Comment:
can you use examples https://github.com/trustbloc/vcs/blob/main/test/bdd/features/vc_v1_e2e_api.feature#L19

Reply:
examples will help not repeat same test you just need pass variables

Reply:
hi @fqutishat , its done now